nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -290,13 +290,71 @@ in
         type = types.bool;
         default = false;
         description = ''
-          Specifies whether sshd(8) should look up the remote host name, and to check that the resolved host name for
+          Specifies whether sshd should look up the remote host name, and to check that the resolved host name for
```

Comment:
That `(8)` is desirable there and can just stay there.

Reply:
Revert